backend/tests/e2e_backend.sh

```diff
@@ -11,7 +11,7 @@ export POD_NAME="e2e"
 
 echo "cleaning up and recreating pod"
 podman pod rm -f ${POD_NAME}
-podman pod create --name=${POD_NAME} --publish 8000:8000
+podman pod create --name=${POD_NAME} --publish 127.0.0.1:8000:8000 --publish 127.0.0.1:3000:3000
```

Comment:
Actually, does this mean that the `local-compose.sh` and `pod-compose.sh` launch scripts need to change, too?

Reply:
I don't really know what `pod-compose.sh` is supposed to do; it's referencing container images I don't believe exist, and somehow I never even noticed that script before.

For testing it's often useful to set up a local pod, and I think `local-compose.sh` is a good idea. The question is whether it still works with your changes. If it needs to be altered, I'd prefer you do that here rather than defer it to later.

Reply:
I wrote those scripts when I created the project years ago, `pod-compose.sh` can go.